antarest/study/storage/rawstudy/raw_study_service.py

```diff
@@ -270,6 +274,7 @@ def build_raw_study(self, dest_name: str, groups: Sequence[str], src_study: Stud
             additional_data=additional_data,
             public_mode=PublicMode.NONE if groups else PublicMode.READ,
             groups=groups,
+            folder=str(Path(destination_folder) / dest_id),
```

Comment:
This is so ridiculous that we need to append the ID :( ... but I know this is the way for consistency with the current mechanisms.

It's safer to not use `Path` here but `PurePosixPath`:
- pure because it does not relate to a real filesystem path
- posix because by convention we use slash separators, even on windows